### PR TITLE
Add blskey:create command - Closes #7180 

### LINKED ISSUE #7180: Add <code class="notranslate">blskey:create` command

### PR DESCRIPTION
**Pull request by Incede**
### What was the problem?

This PR resolves #7180

### How was it solved?

Implement command specified

### How was it tested?

Added unit tests

### CODE REVIEW

**Review by ishantiw — APPROVED** [review]
LGTM

File: commander/test/bootstrapping/commands/blskey/create.spec.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ */
+
+import * as cryptography from '@liskhq/lisk-cryptography';
+import * as Config from '@oclif/config';
+import * as readerUtils from '../../../../src/utils/reader';
+import { CreateCommand } from '../../../../src/bootstrapping/commands/blskey/create';
+import { getConfig } from '../../../helpers/config';
+
+jest.mock('@liskhq/lisk-cryptography', () => ({
+	...jest.requireActual('@liskhq/lisk-cryptography'),
```
[the remark this answers is not in the document]
Comment (Incede) [reply commander/test/bootstrapping/commands/blskey/create.spec.ts:24]: to spy on `cryptography` functions.

[the remark this answers is not in the document]
Comment (shuse2) [reply commander/test/bootstrapping/commands/blskey/create.spec.ts:24]: if it's just to spy
```
jest.spyOn(cryptography, 'generatePrivateKey');
jest.spyOn(cryptography, 'getPublicKeyFromPrivateKey');
```
these without this mock should work?

[the remark this answers is not in the document]
Comment (Incede) [reply commander/test/bootstrapping/commands/blskey/create.spec.ts:24]: It doesn't work. Without the mock it fails with `
    TypeError: Cannot redefine property: generatePrivateKey
        at Function.defineProperty (<anonymous>)` and more errors.

[the remark this answers is not in the document]
Comment (shuse2) [reply commander/test/bootstrapping/commands/blskey/create.spec.ts:24]: mm interesting, maybe problem with how cryptography is exposed. resolving 👍

**Pull request merged**